1_prep/src/munge_nmls.R

```diff
@@ -0,0 +1,51 @@
+#' @title Make nml edits
+#' @description prepare edits that can be made the same way
+#' for all lakes. Applies to one lake's nml list at at time
+#' @param nml_args lake-specific nml parameters loaded in from
+#' the nml_list.rds from lake-temperature-model-prep
+#' @return a list of modified nml arguments
+make_nml_edits <- function(nml_args) {
```

Comment:
ticky tacky, but perhaps a more specific function name would help here since almost all of these are related to adjusting the nml to fit with the maximum depth (or number of hypso layers) of the lake. Perhaps something `adjust_depth_args_nml()` and the `meteo_fl = "NULL"` gets set in the nml template instead of here?

Reply:
Hmm even if I set `meteo_fl="NULL"` in the nml template, it would get over-written in `munge_nmls()` by the `meteo_fl` value stored in the `nml_list.rds` file brought in from `lake-temperature-model-prep`. However, I could move the code to set `meteo_fl` name to NULL into `munge_nmls()` and then I think we could accuratewly rename `make_nml_edits()` to `adjust_depth_args_nml`